docs/io/ascii/index.rst

```diff
@@ -28,9 +28,12 @@ The following shows a few of the ASCII formats that are available, while the sec
 * :class:`~astropy.io.ascii.Rdb`: tab-separated values with an extra line after the column definition line
 * :class:`~astropy.io.ascii.SExtractor`: `SExtractor format table <http://www.astromatic.net/software/sextractor>`_
 
-The :mod:`astropy.io.ascii` package is built on a modular and extensible class
```

Comment:
This little blurb is not really relevant to most users so I'm proposing replacing with the paragraph below.